### DISCUSSION TITLE
Issue with Concurrent Requests Handling in Flowise with Azure OpenAI

### DISCUSSION THREAD

**Discussion by seonghobae** [FlowiseAI/Flowise#2632] [Q&A]
Hello,

I am currently using Azure OpenAI to implement Chat Azure OpenAI. However, it seems that Flowise is unable to handle concurrent requests from clients. When clients send multiple requests simultaneously, Flowise does not appear to process these requests in parallel. The logs in Langfuse suggest that concurrent requests are not being handled properly.

Is there a recommended configuration or method to resolve this issue? How can I ensure that Flowise can handle concurrent requests effectively?

Thank you.

**Comment by qinmaye** [comment]
Have you resolved this question?
